sdk/attestation/azure-security-attestation/samples/sample_attest_enclave_async.py

```diff
@@ -130,27 +163,30 @@ async def attest_open_enclave_with_draft_policy(self):
             c:[type=="x-ms-sgx-mrsigner"] => issue(type="My-MrSigner", value=c.value);
         };
         """
-        print('Attest Open enclave using ', self.shared_url)
-        print('Using draft policy:', draft_policy)
-        async with self._create_client(self.shared_url) as attest_client:
-            response = await attest_client.attest_open_enclave(
-                oe_report, runtime_data=AttestationData(runtime_data, is_json=False),
-                draft_policy=draft_policy)
-
-            print("Token algorithm", response.token.algorithm)
-            print("Issuer of token is: ", response.value.issuer)
+        print("Attest Open enclave using ", self.shared_url)
+        print("Using draft policy:", draft_policy)
+        async with DefaultAzureCredential() as credential, AttestationClient(
+            self.shared_url,
+            credential,
```

Comment:
nit: this can be on one line
```suggestion
            self.shared_url, credential
```